### DISCUSSION TITLE
How do I get the current version of the app

### DISCUSSION THREAD

**Discussion by lgicc** [riderx/capacitor-updater#329] [Q&A]
If I use getBuiltinVersion I get the buildin version of the app. And if I use getLatest I get the newest version. But how do I get the current version of the App so if its not been updated by CapacitorUpdater yet?
e.g. it automatically catches the buildin version otherwise the current app version from the current running update?

**Comment by khamaileon** [comment]
Isn't that what you're looking for?

https://capgo.app/docs/plugin/api/#current
